### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Koliras**
[DEMO LINK](https://Koliras.github.io/react_static-list-of-todos-js/)

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.jsx [hunk 1]
```diff
@@ -1,4 +1,15 @@
...
-  <>TodoInfo markup</>
-);
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={`TodoInfo ${completed && 'TodoInfo--completed'}`}>
```
Comment (davinnchii) [inline src/components/TodoInfo/TodoInfo.jsx:8]: I suggest using classnames method for conditional class setting

File: src/components/TodoList/TodoList.jsx [hunk 2]
```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
```
Comment (davinnchii) [inline src/components/TodoList/TodoList.jsx:5]: Let's name it todoTask, so it will be very obvious what are we iterating through

**Review by davinnchii — CHANGES_REQUESTED** [review]
GJ, but I suggest some changes

File: src/components/TodoList/TodoList.jsx (hunk 2 above)
Comment (tiserett) [reply src/components/TodoList/TodoList.jsx:5]: todoTask sound weird, don't you think? todo is a task itself, todoTask for me is the same as todoTodo or taskTask :))

File: src/components/UserInfo/UserInfo.jsx [hunk 3]
```diff
@@ -1,4 +1,7 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user.name}
+  </a>
```
Comment (tiserett) [inline src/components/UserInfo/UserInfo.jsx:6]: let's add destructuring here

**Review by tiserett — CHANGES_REQUESTED** [review]

File: src/components/TodoInfo/TodoInfo.jsx [hunk 4]
```diff
@@ -1,4 +1,21 @@
...
+      className={classnames({
+        TodoInfo: true,
+        'TodoInfo--completed': completed,
+      })}
+    >
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user && <UserInfo user={user} />}
+
+    </article>
```
Comment (polosanya) [inline src/components/TodoInfo/TodoInfo.jsx:19]:
```suggestion
      {user && <UserInfo user={user} />}
    </article>
```
Don't leave empty lines between parent and child elements

**Review by polosanya — APPROVED** [review]
👍

**Pull request closed without merging**